src/styles/services-view.less

```diff
@@ -1,6 +1,7 @@
 .services-view {
   .services-item {
     color: @text-color;
+    cursor: pointer;
```

Comment:
You shouldn't need this if you make the element a link. Otherwise it's not keyboard accessible.